### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by gomezd**
* Added missing ports
* Added ip addr note for LB case
* Moved `usage` to `administration`
* Added `troubleshooting` for `WebSockets` on LB

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -21,7 +23,7 @@ Run the command to ssh:
 ssh -l admin -i <your-key>.pem <instance-public-IP-address>
 ```
 
-Note that the default user is `admin`.
+Note that the default user is `admin`
```
Comment (luhkevin) [inline docs/installation.md]: The default user is actually `core`.

Comment (gomezd) [inline docs/installation.md]: Fixed

**Comment by luhkevin** [comment]
👍  no issues from my side

File: docs/collectors.md [hunk 2]
```diff
@@ -60,6 +62,8 @@ docker run -td \
        netsil/collectors:${version}
 ```
 
+> *Note: If you are running behind a load balancer you need to use the private ip address instead of the hostname `${hostname}`*
```
Comment (jarnoux) [inline docs/collectors.md]: emphasis on `private ip address` here too?

Comment (gomezd) [inline docs/collectors.md]: fixed

**Pull request merged**